vips/header.c

```diff
@@ -39,7 +14,7 @@ void remove_metadata(VipsImage *in) {
     gchar ** fields = vips_image_get_fields(in);
 
     for (int i=0; fields[i] != NULL; i++) {
-        if (strncmp(fields[i], VIPS_META_ICC_NAME, 16)) {
+        if (strncmp(fields[i], VIPS_META_ICC_NAME, 16) && strncmp(fields[i], VIPS_META_ORIENTATION, 16)) {
```

Comment:
from some reason I cannot find the definition for VIPS_META_ORIENTATION - please make sure that 16 is the size you want to compare and that 16 is not high.

Reply:
` #define VIPS_META_ORIENTATION "orientation"`

It's just for safety

Reply:
try using:
sizeof(VIPS_META_ICC_NAME), sizeof(VIPS_META_ORIENTATION) instead the magic number 16

Reply:
done